source/opt/module.h

```diff
@@ -94,6 +94,7 @@ class Module {
   // Returns a vector of pointers to type-declaration instructions in this
   // module.
   std::vector<Instruction*> types();
+  std::vector<Instruction*> constants();
```

Comment:
Needs a blurb.  Also, since it's not just a member access (and shouldn't be cached), then this needs a first-letter-capitalized method name like Constants() or GetConstants()